### PR TITLE
Freezing pandas at 1.0.5

### PR DESCRIPTION
**Pull request by williamjameshandley**
# Description

Following #115  which notes that master is now broken on account of the pandas update to 1.1, this PR freezes to pandas to 1.0.5 until #115 is resolved

# Checklist:

- [X] I have performed a self-review of my own code
- [X] My code is PEP8 compliant (`flake8 anesthetic tests`)
- [X] My code contains compliant docstrings (`pydocstyle --convention=numpy anesthetic`)
- [X] New and existing unit tests pass locally with my changes (`python -m pytest`)
- [X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def get_version(short=False):
       url='https://github.com/williamjameshandley/anesthetic',
       packages=find_packages(),
       scripts=['scripts/anesthetic'],
-      install_requires=['numpy', 'scipy', 'pandas', 'matplotlib'],
+      install_requires=['numpy', 'scipy', 'pandas<=1.0.5', 'matplotlib'],
```
Comment (lukashergt) [inline setup.py:34]: You've added the `<=1.0.5` flag to both `requirements*.txt` and to `setup.py`. Should that generally be done? Should we do this for `matplotlib` and `scipy` also?

Comment (williamjameshandley) [reply setup.py]: >You've added the <=1.0.5 flag to both requirements*.txt and to setup.py. Should that generally be done? 

There are [ways round the duplication](https://stackoverflow.com/questions/43658870/requirements-txt-vs-setup-py) inherent in having dependencies in both requirements.txt and setup.py, but they're always very fiddly, and just as liable to human error. For now it's simplest/least confusing for users to maintain both, irritating though that is

>Should we do this for matplotlib and scipy also?

Yes : ) Also synced requirements_extras.txt for compleness

**Comment by williamjameshandley** [comment]
Cheers @lukashergt. You should update #114 accordingly so that we can merge that before more major changes occur.

Also just occurred to me how odd the title of this PR would look to somebody not familiar with the python datascience ecosystem.

**Comment by lukashergt** [comment]
> Also just occurred to me how odd the title of this PR would look to somebody not familiar with the python datascience ecosystem.

So true! Hadn't thought of that. Haha, thanks for that!

**Pull request merged**